### PR TITLE
4 install docker

### LINKED ISSUE #4: Sean - Install and configure Docker on your PC

### PR DESCRIPTION
**Pull request by bushidocodes**
Resolves #4

### CODE REVIEW

File: node.js [hunk 1]
```diff
@@ -666,6 +666,7 @@ async function stabilize() {
 
 /* added 20191021 */
 async function stabilize_self() {
+    console.log("Here is a terrible bug");
```
Comment (heidiklum) [inline node.js:669]: this should be after the documentation lines for the method.  suggest it be moved down by five lines.

File: setup-docker.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Setup Docker
```
Comment (heidiklum) [inline setup-docker.md:1]: thanks for the informative comment right at the beginning.

File: node.js (hunk 1 above)
Comment (bushidocodes) [reply node.js:669]: WUT U TALKING BOUT? ADD MY CODE. GRR.

**Review by heidiklum — CHANGES_REQUESTED** [review]
maybe we should leave this off for now?

**Pull request closed without merging**